src/components/help/help.pw.tsx

```diff
@@ -0,0 +1,354 @@
+    });
+  });
+
+  testData.forEach((children) => {
+    test(`should check children as ${children}`, async ({ mount, page }) => {
+      await mount(<Help> {children} </Help>);
+      await checkAccessibility(page);
+    });
+  });
+
+  testData.forEach((helpId) => {
+    test(`should check id as ${helpId}`, async ({ mount, page }) => {
+      await mount(<HelpComponentTest helpId={helpId} />);
+      await checkAccessibility(page);
+    });
+  });
+
+  ["https://carbon.sage.com", "google", "https://www.google.com"].forEach(
+    (hrefLink) => {
+      test(`should check href link as ${hrefLink}`, async ({ mount, page }) => {
+        await checkAccessibility(page);
+      });
+    }
+  );
+
+  [true, false].forEach((boolVal) => {
+    test(`should check when isFocused prop is passed as ${boolVal}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest isFocused={boolVal}>{tooltipText}</HelpComponentTest>
+      );
+      await checkAccessibility(page);
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipBgColor as ${names}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipBgColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page, "color-contrast");
+    });
+  });
+
+  colors.forEach(([names, color]) => {
+    test(`should check tooltipFontColor as ${names}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest tooltipFontColor={color} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page, "color-contrast");
+    });
+  });
+
+  (["top", "bottom", "left", "right"] as HelpProps["position"][]).forEach(
+    (position) => {
+      test(`should render tooltipFlipOverrides as ${position}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(
+          <HelpComponentTest tooltipFlipOverrides={[position]} isFocused>
+            {`This tooltip is positioned ${position}`}
+          </HelpComponentTest>
+        );
+        await checkAccessibility(page, "color-contrast");
+      });
+    }
+  );
+
+  testData.forEach((tooltipId) => {
+    test(`should check tooltipId as ${tooltipId}`, async ({ mount, page }) => {
+      await mount(
+        <HelpComponentTest tooltipId={tooltipId} isFocused>
+          {tooltipText}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page);
+    });
+  });
+
+  (["top", "bottom", "left", "right"] as any[]).forEach((tooltipPosition) => {
+    test(`should render with tooltip positioned ${tooltipPosition}`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(
+        <HelpComponentTest tooltipPosition={tooltipPosition} isFocused>
+          {`This tooltip is positioned ${tooltipPosition}`}
+        </HelpComponentTest>
+      );
+      await checkAccessibility(page);
```

Comment:
**suggestion: (non-blocking)** as above, it would be better to add a space to separate the mount from the assertion to follow the same pattern as with the other test cases.